internal/locale/locales/en-us.yaml

```
@@ -1436,11 +1436,11 @@ deploy_cmd_description:
 deploy_install_cmd_description:
   other: Only install the runtime environment files, but do not configure it in any way (effectively caches the runtime environment)
 deploy_configure_cmd_description:
-  other: Configure the runtime environment for your current shell (must have ran install first or this will err)
+  other: Configure the runtime environment for your current shell (must have run install first or this will err)
```

While we're here can we change this as well?

```suggestion
  other: Configure the runtime environment for your current shell (must have run install first or this will error)
```